services/brig/src/Brig/API/Public.hs

```diff
@@ -305,6 +307,57 @@ type ListClientsBulk =
     :> Servant.ReqBody '[Servant.JSON] (Range 1 MaxUsersForListClientsBulk [Qualified UserId])
     :> Post '[Servant.JSON] (Public.QualifiedUserMap (Set Public.Client))
 
+type GetUsersPrekeysClientUnqualified =
+  Summary "(deprecated) Get a prekey for a specific client of a user."
+    :> "users"
+    :> CaptureUserId "uid"
+    :> "prekeys"
+    :> CaptureClientId "client"
+    :> Get '[Servant.JSON] Public.ClientPrekey
+
+type GetUsersPrekeysClientQualified =
+  Summary "Get a prekey for a specific client of a user."
+    :> "users"
+    :> Capture "domain" Domain
+    :> CaptureUserId "uid"
+    :> "prekeys"
+    :> CaptureClientId "client"
+    :> Get '[Servant.JSON] Public.ClientPrekey
+
+type GetUsersPrekeyBundleUnqualified =
+  Summary "(deprecated) Get a prekey for each client of a user."
+    :> "users"
+    :> CaptureUserId "uid"
+    :> "prekeys"
+    :> Get '[Servant.JSON] Public.PrekeyBundle
+
+type GetUsersPrekeyBundleQualified =
+  Summary "Get a prekey for each client of a user."
+    :> "users"
+    :> Capture "domain" Domain
+    :> CaptureUserId "uid"
+    :> "prekeys"
+    :> Get '[Servant.JSON] Public.PrekeyBundle
```

Comment:
The response type doesn't contain the domain, so clients have to stitch requrest+response together, which is okay I think, wdyt? Note that for example /usr/:uid/prekeys also doesn't contain the user in the response.

Reply:
decided not to include it